<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finvoicevalinnat</class>
 <widget class="QWidget" name="Finvoicevalinnat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="kaytossaGroup">
     <property name="title">
      <string>Finvoice-verkkolaskut</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="eiKaytossa">
        <property name="text">
         <string>Ei käytössä</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="paikallinen">
        <property name="text">
         <string>Verkkolaskujen paikallinen muodostaminen tiedostoon</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QRadioButton" name="integroitu">
          <property name="text">
           <string>Integroitu verkkolaskutus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ivAsetusNappi">
          <property name="text">
           <string>Asetukset...</string>
          </property>
          <property name="icon">
           <iconset resource="../../pic/pic.qrc">
            <normaloff>:/pic/ratas.png</normaloff>:/pic/ratas.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="osoiteGroup">
     <property name="title">
      <string>Oma verkkolaskuosoite</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Verkkolaskuosoite</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ovtEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Välittäjän tunnus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="operaattoriEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="hakemistoGroup">
     <property name="title">
      <string>Paikalliset asetukset</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ohjelma muodostaa verkkolaskuja, mutta käyttäjän on itse ladattava ne verkkolaskuoperaattorille</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Tallennushakemisto</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QLineEdit" name="hakemistoEdit">
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hakemistoNappi">
            <property name="text">
             <string>Valitse hakemisto</string>
            </property>
            <property name="icon">
             <iconset resource="../../pic/pic.qrc">
              <normaloff>:/pic/folder.png</normaloff>:/pic/folder.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="noutoGroup">
     <property name="title">
      <string>Verkkolaskujen noutaminen</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="eiVastaanottoa">
        <property name="text">
         <string>Verkkolaskujen vastaanotto ei käytössä</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="saapuneetKansio">
        <property name="text">
         <string>Nouda ohjelmaa käynnistettäessä</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="kiertoRadio">
          <property name="text">
           <string>Lisää kiertoon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kiertoCombo"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="postitusCheck">
     <property name="text">
      <string>Käytä postituspalvelua</string>
     </property>
     <property name="icon">
      <iconset resource="../../pic/pic.qrc">
       <normaloff>:/pic/kirje.png</normaloff>:/pic/kirje.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>171</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
